reference/res/res.send.md

```diff
@@ -1,58 +1,35 @@
 # res.send()
 
-Send a simple response.  `statusCode` defaults to 200 ("OK").
+This method us used to send a string as a response.
```

Comment:
### Let's keep it in the imperative mood

Send a string response in a format other than JSON (XML, CSV, plain text, etc.)